pages/news.tsx

```diff
@@ -1,5 +1,5 @@
 import Image from "next/image";
-import useFetchNews from "./hooks/useFetchNews";
+import useFetchNews from "../hooks/useFetchNews";
 import { poppins } from "../styles/fonts";
```

Comment:
I am not expecting this here

Reply:
You can (and should) create a layout component that imports all your fonts and attaches them to the DOM.

Not like this